definitions/README.md

```diff
@@ -2,3 +2,84 @@
 
 This directory contains definitions for specific metrics, data sources, dimensions or segments for the platform they target.
 These definitions can be referenced in project specific configuration files without having to be redefined.
+
+Metric definitions can be validated using the following Python script in Google Colab:
+
+First, ensure that `mozanalysis` is available:
+
+```shell
+!pip install mozanalysis -qq
+```
+
+Second, setup the environment:
+
+```python3
+import pandas as pd
+
+from google.cloud import bigquery
+from google.colab import auth
+from mozanalysis.config import ConfigLoader
+from textwrap import dedent
+
+auth.authenticate_user()
+
+def fetch_metric_hub_definition(start_date, end_date, metric_slug, app_name, bq_project):
```

Comment:
What do you think about a name like `fetch_validation_dataframe`? When I think of a metric "definition" I think of metadata, whereas this function returns materialized data.